.travis.yml

```diff
@@ -94,6 +92,12 @@ before_install:
 
       # Qt 5
       PATH=/opt/qt55/bin:${PATH}
+
+      if [ "$build_system" = "cmake" ]; then
+          COMPILER_VERSION=6
+          sudo update-alternatives --install /usr/bin/${CC} ${CC} /usr/bin/${CC}-${COMPILER_VERSION} 60 --slave /usr/bin/${CXX} ${CXX} /usr/bin/${CXX}-${COMPILER_VERSION}
+          export CXX="${CXX}-${COMPILER_VERSION}" CC="${CC}-${COMPILER_VERSION}"
```

Comment:
IIRC, using export CXX & CC is enough, you don't need `update-alternatives`.
Then no sudo is required.